Dockerfile

```diff
@@ -18,7 +18,7 @@
 # TODO: verify commit hash
 
 # Basic deps
-ARG CONTAINERD_VERSION=v1.7.3
+ARG CONTAINERD_VERSION=v1.7.6
```

Comment:
I guess we need to introduce this version in the CI matrix to ensure support

Reply:
I already did in https://github.com/containerd/nerdctl/commit/e4a2ceb72895345b2a1b1e9ab8d8dad54fbd385e but forgot to update Dockerfile